src/components/BooksForm.js

```diff
@@ -33,25 +33,36 @@ function Form() {
   };
 
   return (
-    <form className="FORM" onSubmit={handleSubmit}>
-      <input
-        type="text"
-        id="title"
-        name="title"
-        required
-        value={state.title}
-        onChange={onChangeHandler}
-      />
-      <input
-        type="text"
-        id="author"
-        name="author"
-        required
-        value={state.author}
-        onChange={onChangeHandler}
-      />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="Add-container">
+      <h2 className="add-title">Add Your Book</h2>
```

Comment:
- [ ]  👍 Great job with the color selection! To make your design even closer to the intended look, kindly consider updating the font family to match the design. This will ensure a consistent and visually appealing layout. Keep up the good work! 👍🎨